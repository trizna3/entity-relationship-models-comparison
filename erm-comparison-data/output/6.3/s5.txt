teams-teams
rounds-rounds
matches-matches
players-players
points-points
