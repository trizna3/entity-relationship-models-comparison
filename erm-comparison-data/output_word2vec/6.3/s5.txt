teams-teams
matches-matches
points-points
players-players
rounds-rounds
