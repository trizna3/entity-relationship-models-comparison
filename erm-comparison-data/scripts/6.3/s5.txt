entity set teams {
};

entity set players {
};

entity set matches {
};

entity set rounds {
};

entity set points {
	"goal scored"
	"assistance"
	"period"
	"minute"
};

association between 
{
	teams ONE
	matches MANY
}{};

association between 
{
	teams ONE
	matches MANY
}{};

association between 
{
	matches MANY
	rounds ONE
}{};

association between 
{
	teams ONE
	players MANY
}{};

association between 
{
	players ONE
	points MANY
}{};

association between 
{
	matches ONE
	points MANY
}{};