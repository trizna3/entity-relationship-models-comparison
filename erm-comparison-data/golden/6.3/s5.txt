teams-teams
players-players
matches-matches
points-points
rounds-rounds
